stock_inventory_valuation_report/wizard/stock_quantity_history.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class StockQuantityHistory(models.TransientModel):
+    _inherit = "stock.quantity.history"
+
+    compute_at_date = fields.Selection(
+        [("0", "Current Inventory"), ("1", "At a Specific Date")],
+        string="Compute",
+        help="Choose to analyze the current inventory or from a specific date in the past.",
+        default="0",
+    )
+    inventory_datetime = fields.Datetime(
+        "Inventory at Date",
+        help="Choose a date to get the inventory at that date",
+        default=fields.Datetime.now,
+    )
```

Comment:
**issue**: I still don't understand why there's a need to re-declare these fields instead of using the fields that already exist.

Cf. the module on v12 that we are porting to v14: https://github.com/OCA/stock-logistics-reporting/blob/12.0/stock_inventory_valuation_report/wizard/stock_quantity_history.py